docs/testing.md

```diff
@@ -0,0 +1,17 @@
+# How to test
```

Comment:
I thought we wanted to keep the wiki for documentation?

Reply:
In a phone call: "Olly, in case you find time, you may split up the wiki to docs and user documentation". Plan: Have the content hosted at https://readthedocs.org/ and enable PR review on documentation improvements.

Refs https://github.com/JabRef/user-documentation/issues/178